src/components/EntryEditor/__tests__/__snapshots__/EntryEditorToolbar.spec.js.snap

```diff
@@ -1,3 +1,5 @@
-exports[`EntryEditorToolbar should disable and update label of Save button when persisting 1`] = `"<div><button disabled=\"\" class=\"\" type=\"button\" data-react-toolbox=\"button\">Saving...</button> <button class=\"\" type=\"button\" data-react-toolbox=\"button\">Cancel</button></div>"`;
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
These snapshot changes shouldn't be a part of this PR - it looks like these are due to either the `jest` update above or a previous commit not updating the snapshots when they should have been. Either way, this is unrelated to the rest of the PR, and should probably be handled separately.